src/Symfony/Component/Validator/Resources/translations/validators.id.xlf

```diff
@@ -36,11 +36,11 @@
             </trans-unit>
             <trans-unit id="9">
                 <source>This field was not expected.</source>
-                <target>Bidang ini tidak diharapkan.</target>
```

Comment:
@IndraGunawan thanks for the reference! The mentioned official translation list is clearly better than the automatic Google translation, so 👍 to those changes too. Thanks!